### PR TITLE
Remove cyclic dependency

### PR DESCRIPTION
**Pull request by bamachrn**
This PR
* removed multiple dependencies from the entries
* this makes sure the image are not getting triggered multiple times, in case of centos base image update

### CODE REVIEW

File: index.d/index_template.yaml [hunk 1]
```diff
@@ -20,8 +20,6 @@ Projects:
 # If you have any dependant containers use this
 # Please check the other yml files for ids that are free
 #
-#   depends-on:
-#     - namespace1/container_name1:tag1
-#     - namespace2/container_name2:tag2
+#   depends-on: namespace1/container_name1:tag1
 #
-# Just copy paste for multiple entries
+# Just copy paste immedieate dependecy entry
```
Comment (navidshaikh) [inline index.d/index_template.yaml]: *immediate
*dependency

**Comment by bamachrn** [comment]
#dotests

File: index.d/index_template.yaml (hunk 1 above)
Comment (bamachrn) [reply index.d/index_template.yaml]: fixed

File: index.d/che-stacks.yaml [hunk 2]
```diff
@@ -130,15 +130,15 @@ Projects:
...
-    job-id: centos-quarkus-io-jdk8
-    git-url: https://github.com/eclipse/che-dockerfiles
-    git-branch: master
-    git-path: recipes/centos_quarkus.io_jdk8
-    target-file: Dockerfile
-    desired-tag: latest
-    notify-email: mike@vorburger.ch
-    build-context: ./
-    depends-on: che-stacks/centos-jdk8:latest
+#
```
Comment (navidshaikh) [inline index.d/che-stacks.yaml]: Please add a comment here as TODO about why we commented these lines and next actions to be taken.

Comment (bamachrn) [reply index.d/che-stacks.yaml]: done

**Pull request merged**